### PR TITLE
Fix for appending null req vars

### PR DESCRIPTION
**Pull request by andralungu**
Earlier today, an issue was opened saying that:
`failed to run log_by_lua*: ...y/lualib/api-gateway/tracking/RequestTrackingManager.lua:242: attempt to concatenate local 'v' **(a nil value)**`

This is a quick and dirty fix for it. 
@ddragosd, @cristianconstantin ping.

### CODE REVIEW

File: src/lua/api-gateway/tracking/RequestTrackingManager.lua [hunk 1]
```diff
@@ -229,16 +229,18 @@ local function matchVarsWithDomains(vars, domains, cache, separator)
         -- ngx.log(ngx.DEBUG, "VAR " , tostring(i))
         local variableManager = ngx.apiGateway.tracking.variableManager
         v = variableManager:getRequestVariable(vars[i], cache)
-        -- ngx.log(ngx.DEBUG, "VAL=", v)
-        d = domains[i]
-        -- ngx.log(ngx.DEBUG, "DOMAIN=", d)
-        if (d == "*") then
-            str = str .. v .. separator
```
Comment (ddragosd) [inline src/lua/api-gateway/tracking/RequestTrackingManager.lua:236]: @andralungu what if we consider instead:

```
str = str .. tostring(v) .. separator
```

In this way the module is indeed returning all the values, even when `v` is nil, which is happening when the NGINX variable is not defined.

**Comment by andralungu** [comment]
@ddragosd that would also work :). Sorry the slight delay in making the adjustment.

**Comment by ddragosd** [comment]
👍

**Comment by ddragosd** [comment]
thanks @andralungu

**Pull request merged**